src/EFCore.Jet/EFCore.Jet.csproj

```diff
@@ -1,23 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net461;netcoreapp3.1</TargetFrameworks>
+    <TargetFramework>$(DefaultNetStandardTargetFramework)</TargetFramework>
```

Comment:
I know it may not seem necessary, but I'd be inclined to at least dual target .NET Standard 2.0 and .NET Core 3.1.

Does EFCore.Jet work on any .NET Standard 2.0 platform other than .NET Framework and .NET Core 3.1?  My recollection is that it will not run properly on .NET Core 2.x (even though those platforms may be compatible with .NET Standard 2.0).  I feel like if people using .NET Core 2.x at least see .NET Core 3.1 listed in the dependencies section of the NuGet package, they could be led to think, "Hey, maybe I should try upgrading my project to .NET Core 3.1 to use this library" rather than seeing .NET Standard 2.0 and assuming that it should work.

Reply:
Our only real dependencies are EF Core 3.1.x (which is .NETStandard 2.0 compliant in contrast to EF Core 3.0.x which was .NETStandard 2.1 compliant) and currently `System.Data.OleDb`, which also is .NETStandard 2.0 compliant). If we should switch to ODBC in the future (or implement additional support for it), then they are also .NETStandard 2.0 compliant. So as long as an app runs on a platform that provides an OLEDB or ODBC driver for Jet, there should be no other restrictions.

That means for example, that it is absolutely possible to compile a .NET Core 2.2.x app with EF Core 3.1.x and EFCore.Jet 3.1.x. It will also be possible to run a .NET Framework 4.7.2 app with EF Core 3.1.x and EFCore.Jet 3.1.x.

Reply:
Yes, I am aware that it will compile.  That's my problem with it.  It will compile, but my recollection is that it won't actually run on all .NET Standard platforms, especially .NET Core 2.x (I haven't tested with this particular code, of course, but that's been my experience with others).

I realize (like you say) that System.Data.OleDb has largely the same issue, and yet they still target .NET Standard 2.0, so keeping the dependency the same makes sense from that perspective, and it keeps maximum compatibility.  However, what I've seen other NuGet packages do is still target  .NET Standard 2.0 (the only dependency needed for compiling), but also specifically target .NET Core 3.1, .NET Framework 4.7.2, or other specific versions.  This makes fixing bugs that only occur on a single platform easier, as well as gives people consuming the NuGet package an idea of what frameworks the authors actually designed it for (which can be SUPER helpful as a consumer of the package).

I don't think that it is such a serious issue to block merging the PR or anything, but I do think that multi-targeting the way that many other NuGet packages can be really helpful.

Reply:
I don't see how this is related to easier platform dependent bug fixing. However, I think you are making a good point, that explicitly specifying intended (or minimal supported) platforms will aid users to do the right thing. That is an important aspect I did not think about.
I will update the target frameworks for `EFCore.Jet` and `System.Data.Jet` accordingly.

Reply:
Yes, my terminology is bad.  I don't mean platform-dependent bugs so much as runtime-dependent bugs, where you can use `#if NETCOREAPP` and `#if NETFRAMEWORK`

Reply:
Also, as a minor point, .NET 5 will supposedly allow us to target `net5.0-win`, which will allow us to forbid Linux/OSX.  Although, I think we might have to drop support for .NET Standard 2.0 to completely eliminate those platforms.

Reply:
I added .NET Framework 4.6.1 and .NET Core 2.1 to hint at the minimum supported framework versions.